CHANGELOG.md

```diff
@@ -9,6 +9,19 @@
 * `RAW_TEXT` and `TEXT_FILE` attachment types to replace the `TEXT` type.
 
 # Version 3.25.3 (2022-08-10)
+
+### Added
+* `ModelRun.update_config()`
```

Comment:
The changelogs for these functions should go in a new section above version 3.26.0. We are now modifying changelogs each commit, and best practice is to add a new placeholder version at the top, like so:
```
# Version 0.0.0 (YYYY-MM-DD)
## Added
* `ModelRun.update_config()`
...
```